src/ASM/ASMs2D.C

```diff
@@ -1365,6 +1365,8 @@ bool ASMs2D::getElementCoordinates (Matrix& X, int iel) const
 #endif
 
   const Go::SplineSurface* geo = this->getBasis(ASM::GEOMETRY_BASIS);
+  if (forceItg)
+    geo = surf;
```

Comment:
Isn't it better with `const Go::SplineSurface* geo = forceItg ? surf : this->getBasis(ASM::GEOMETRY_BASIS);` such that `getBasis()` is invoked only when needed?

Reply:
sure, i guess we can eat the long line for the minor optim.